internal/app/recordtester/recordtester_app.go

```diff
@@ -155,14 +121,9 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 	// glog.Infof("All cool!")
 	hostName, _ := os.Hostname()
 	streamName := fmt.Sprintf("%s_%s", hostName, time.Now().Format("2006-01-02T15:04:05Z07:00"))
-	var stream *livepeer.CreateStreamResp
+	var stream *api.CreateStreamResp
 	for {
-		stream, err = rt.lapi.CreateStream(livepeer.CreateStreamReq{
-			Name:                streamName,
-			Profiles:            standardProfiles,
-			Record:              true,
-			RecordObjectStoreId: rt.recordObjectStoreId,
-		})
+		stream, err = rt.lapi.CreateStreamEx2(streamName, true, rt.recordObjectStoreId, "", nil, api.StandardProfiles...)
```

Comment:
OK I saw this and decided to pay my debts on the `go-api-client` repo by properly upstreaming the `CreateStream` changes there: https://github.com/livepeer/go-api-client/pull/10

If you're OK with it, mind reviewing that one so I can merge and here you can use the same `CreateStream` function that was used before?